README.md

```diff
@@ -30,10 +30,24 @@
 -->
 ```html
 <vaadin-notification opened position="middle" duration="-1">
-  <template>
-    Your work has been saved
-  </template>
 </vaadin-notification>
+
+<script>
+  const notification = document.querySelector('vaadin-notification');
+
+  notification.renderer = function(root) {
+    // Check if there is a content generated with the previous renderer call not to recreate it.
```

Comment:
This is overly complex, the original template only had text so this renderer should only do `root.textContent = 'Your work has been saved'`!!! Same thing  for the other file

Reply:
Fixed.